Sources/Fuzzilli/FuzzIL/Instruction.swift

```diff
@@ -590,6 +592,9 @@ extension Instruction: ProtobufConvertible {
             op = Await()
         case .callMethod(let p):
             op = CallMethod(methodName: p.methodName, numArguments: inouts.count - 2)
+        case .callComputedMethod(_):
+            // We subtract 3 from the inouts count since the first two elements are the callee and method and the last element is the instruction index
```

Comment:
I think this comment isn't correct. The first two are the reference object and the method, and the last element is the output variable (not the instruction index)